lib/rucio/common/schema/generic.py

```diff
@@ -187,6 +187,24 @@
     "pattern": r'^((25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)(\.|$)){4}$'
 }
 
+IPv4orIPv6 = {
+    "description": "IPv4 or IPv6 address",
+    "type": "string",
+    "pattern": r'((^\s*((([0-9]|[1-9][0-9]|1[0-9]'
+    r'{2}|2[0-4][0-9]|25[0-5])\.){3}([0-9]|[1-9][0-9]|1[0-9]{2}|2[0-4][0-9]|25[0-5]))\s*$)|'
```

Comment:
<troll mode> I think there is a bug on character 143. </troll mode> 
But, to be honest, this is impossible to review and it can even be compiled to use bactrack, which would be terrible for performance. Isn't there any way to call a function instead of using a pattern for validation ?  Then we can rely on https://docs.python.org/3/library/ipaddress.html#ipaddress.ip_address for validation.

Reply:
Well, there are even native ip formats :) https://python-jsonschema.readthedocs.io/en/stable/validate/#validating-formats

Reply:
Yes, please change this Mayank. No need to put commonly defined formats to the schema.